src/app/collections/reservation_list.js

```diff
@@ -0,0 +1,16 @@
+import Backbone from 'backbone';
+import Reservation from '../models/reservation';
+
+const ReservationList = Backbone.Collection.extend({
+  model: Reservation,
+
+  url: function() {
+    // return `https://ada-backtrek-api.herokuapp.com/trips/${this.id}/reservations`
```

Comment:
This could work, if you had modified it to do:

`return https://ada-backtrek-api.herokuapp.com/trips/${this.get('trip_id')}/reservations`